abstractions/dotnet/src/IAuthenticationProvider.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+using System.Threading.Tasks;
+
+namespace Kiota.Abstractions {
+    public interface IAuthenticationProvider {
+        Task<string> getAuthorizationToken(Uri requestUri);
```

Comment:
Public interface member should be in PascalCase.
 ```suggestion
         Task<string> GetAuthorizationToken(Uri requestUri);
```

Reply:
fixed casing